### PR TITLE
Create an operational model for pthread_equal

### PR DESCRIPTION
**Pull request by lucasccordeiro**
This PR aims to create an operational model `pthread_equal` and also instructions on how to create pull requests for ESBMC.

### CODE REVIEW

**Comment by lucasccordeiro** [comment]
@fatimahkj: May I ask you to review this PR?

**Review by fatimahkj — APPROVED** [review]

**Review by fatimahkj — APPROVED** [review]

**Review by fatimahkj — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -80,6 +80,38 @@ Here are some steps to contributing to the code base:
...
+
+Note that you can check your changes via `git status`. 
+Note also that every PR should contain at least two test cases to check your implementation: one successful and one failed test
+ case.
+
+D) Push your changes in the local branch to the ESBMC repository:
+
+$git push origin model-pthread-equal
+```
+
```
Comment (rafaelsamenezes) [inline README.md:114]: Although this shouldn't belong in this PR I see no problem in merging it.

**Review by rafaelsamenezes — APPROVED** [review]
LGTM.

File: README.md (hunk 1 above)
Comment (lucasccordeiro) [inline README.md:114]: I agree with you.

I have created this section in the README to help new contributors as @fatimahkj to engage with our research group.
